client/app/lib/util/routeHandler.coffee

```diff
@@ -29,6 +33,51 @@ module.exports = (options) ->
       app.openSection section, query, action, identifier
 
 
+  handleGitLab = (options, path) ->
+
+    { params } = options
+    { section, action, identifier, extra, branch } = params
+
+    repo = "#{identifier}/#{extra}"
+    repo = "#{repo}/#{branch}"  if branch
```

Comment:
we don't like concatenation here https://github.com/koding/styleguide-coffeescript#use-string-interpolations-instead-of-string-concatenation :)